Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/model/ContactDiaryLocationVisit.kt

```diff
@@ -0,0 +1,12 @@
+package de.rki.coronawarnapp.contactdiary.model
+
+import org.joda.time.LocalDate
+
+interface ContactDiaryLocationVisit {
+    val id: Long
+    val date: LocalDate
+    val contactDiaryLocation: ContactDiaryLocation
```

Comment:
Remark: wouldn't `location` be enough?

Reply:
"Location" is a usual datatype in Android/Kotlin, containing Lat/Lng.. To avoid confusion for the reader I prefer more specified naming, like contactDiaryLocation/Person, that anyone knows the referings..

Reply:
Fair enough